src/mscorlib/src/System/Threading/Thread.cs

```diff
@@ -212,7 +201,7 @@ static void AsyncLocalSetCurrentUICulture(AsyncLocalValueChangedArgs<CultureInfo
             SetStartHelper((Delegate)start, 0);
         }
 
-        public Thread(ParameterizedThreadStart start, int maxStackSize) {
+        internal Thread(ParameterizedThreadStart start, int maxStackSize) {
```

Comment:
I'm surprised these public methods are being removed; aren't we exposing them in .NET Standard 2.0?